styles.css

```diff
@@ -83,7 +84,198 @@
-  border: 0.5px solid #646369;
+.horizontal-line {
+  border: 0.2px solid #646369;
   margin: 25px;
 }
+
+.about-section {
+  display: grid;
+  min-height: 40vh;
+  grid-template-columns: auto auto;
+  padding: 1rem;
+  gap: 10px;
+  background: url("./assets/images/ellipse5.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.skill-set {
+  background-color: transparent;
+}
+
+.title {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  color: #9490ff;
+  padding: 0.6rem 1.2rem 0.6rem 1.2rem;
+  background-color: transparent;
+}
+
+.skills {
+  list-style-type: none;
+  padding: 1.2rem;
+  color: white;
+  font-size: 15px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 24px;
+  background-color: transparent;
+}
+
+.skills > li {
+  margin-bottom: 0.3rem;
+  background-color: transparent;
+}
+
+.social-icons > ul {
+  min-height: 10vh;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.social-icons > ul > li > a {
+  color: white;
+  padding: 5px;
+  zoom: 1.5;
+  text-decoration: none;
+}
+
+.work-section {
+  display: grid;
+  min-height: 50vh;
+}
+
+.work-section-title-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  color: #a2a3ae;
+}
+
+.work-section-title {
+  text-align: center;
+  color: white;
+  padding: 10px;
+  font-size: 20px;
+  font-style: normal;
+  line-height: 32px;
+  font-weight: 500;
+}
+
+.work-icon {
+  zoom: 2;
+}
+
+.recent-work {
+  color: #9490ff;
+  font-size: 50px;
+  line-height: 60px;
+  font-style: normal;
+  font-weight: 500;
+  padding: 60px 0 60px 1.7rem;
+}
+
+.work-section-hr {
+  margin-top: -60px;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+}
+
+.project-image {
+  width: 340px;
+  height: 236px;
+  background-color: transparent;
+}
+
+.project-image-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: transparent;
+}
+
+.project-info-container1 {
+  background: url("./assets/images/ellipse1.svg");
+  background-repeat: no-repeat;
+}
+
+.project-info-container2 {
+  background: url("./assets/images/ellipse2.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.project-info-container3 {
+  background: url("./assets/images/ellipse3.svg");
+  background-repeat: no-repeat;
+}
+
+.project-info-container4 {
+  background: url("./assets/images/ellipse4.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.project-name {
+  background-color: transparent;
+  padding: 2rem;
+  color: white;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.project-description {
+  font-weight: 400;
+  font-style: normal;
+  line-height: 28px;
+  font-size: 20px;
+  padding: 2rem;
+  color: white;
+  font-family: Inter -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+  background-color: transparent;
+}
```

Comment:
- [ ] The correct font is still not been applied properly as your page font still looks different from the Figma template. The correct font family for this `p` is `Inter`. Please consider adding this font family to this paragraph like this: `font-family: 'Inter', sans-serif;`
- [ ] Also add the correct color `color: #DCDCE8;`

Reply:
Applied correct font family and color to project-description class